server/models/Trips.js

```diff
@@ -68,6 +66,16 @@ class Trips {
 		return this.result;
 	}
 
+	static async viewActivetrip() {
+		// eslint-disable-next-line radix
+		const obj = db.find(o => o.status === 'active');
+		if (!obj) {
+			return false;
+		}
+		this.result = obj;
```

Comment:
If this method is static, why are you using the keyword "this"?

Reply:
on the constructor of the class. I have assigned this.result  to be null. When I want to return the queried data. I just pass the data into "this.result" and return it to the user as a response.

Reply:
What I mean is, this method can be used without instantiating a class which means, "this" will be undefined, since "this" refers to an instance of the class

Reply:
I would recommend that you don't use the "static" key word

Reply:
let me push the changes and update the methods from been 'static'